user_guide.adoc

```diff
@@ -412,15 +412,14 @@ seq:
 
 Of course, you can use any other byte (for example, `0xa`, AKA
 newline) as a terminator. This gives Kaitai Struct some limited
-capabilities to parse of certain text formats as well.
+capabilities to parse certain text formats as well.
 
 Reading "until the terminator byte is encountered" could be
-dangerous. What if we'll never encounter that byte?
+dangerous. What if we never encounter that byte?
 
 Another very widespread model is actually having *both* a fixed-sized
 buffer for a string *and* a terminator. This is typically an artifact
-of serializing structures like this from C. For example, if one would
-just take this example structure:
+of serializing structures like this from C. For example, if one would take this example structure:
```

Comment:
Keep the row width:

```suggestion
of serializing structures like this from C. For example, if one would
take this example structure:
```

Reply:
Thanks for spotting this. I have resolved this in a different way.